dns-policy-operator/charmcraft.yaml

```diff
@@ -0,0 +1,61 @@
+name: dns-policy
+type: charm
+title: DNS policy charm
+
+summary: A very short one-line summary of the charm.
+
+description: |
+bases:
+  - build-on:
+    - name: ubuntu
+      channel: "22.04"
+    run-on:
+    - name: ubuntu
+      channel: "22.04"
+
+config:
+  options:
+    # An example config option to customise the log level of the workload
+    log-level:
+      description: |
+        Configures the log level of gunicorn.
+        Acceptable values are: "info", "debug", "warning", "error" and "critical"
+      default: "info"
+      type: string
+    debug:
+      type: boolean
+      description: "Whether dns-policy-app debug mode is enabled."
+      default: false
+    allowed-hosts:
+      type: string
+      description: A comma-separated list of host/domain names that the dns-policy-app API
+        can serve. This configuration will set the DJANGO_ALLOWED_HOSTS environment
+        variable with its content being a JSON encoded list.
+      default: "localhost, 127.0.0.1, 0.0.0.0"
+
+requires:
+  database:
+    interface: postgresql_client
+    limit: 1
+  dns-record-requirer:
+    interface: dns_record
+    scope: container
+    limit: 1
+
+provides:
+  dns-record-provider:
+    interface: dns_record
+
+parts:
+  app:
+    plugin: dump
+    source: .
+    prime:
+      - dns-policy-app_*_amd64.snap
```

Comment:
As discussed I suppose this will be removed, it's used for dev purpose for now ?

Reply:
This will be removed in a future PR, after https://github.com/canonical/dns-operators/pull/91 that will enable the necessary publishing workflows